flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/ProcessVariableUpdateAction.java

```diff
@@ -0,0 +1,29 @@
+package com.flash3388.frc.nt;
+
+import com.flash3388.flashlib.scheduling.actions.ActionBase;
+
+import java.util.function.DoubleConsumer;
+import java.util.function.DoubleSupplier;
+
+public class ProcessVariableUpdateAction extends ActionBase {
+
+    private final DoubleSupplier processVariableSupplier;
+    private final DoubleConsumer processVariableConsumer;
+    private double prevValue;
+
+    public ProcessVariableUpdateAction(DoubleSupplier processVariableSupplier, DoubleConsumer processVariableConsumer) {//do I need to receive the scheduler from the user?
```

Comment:
Instead of commenting in your code, just comment in the PR. I might miss it or you might forget to delete it.

Reply:
yeah all right